universal-ctags.rb

```diff
@@ -1,17 +1,24 @@
 class UniversalCtags < Formula
   homepage 'https://github.com/universal-ctags/ctags'
   head 'https://github.com/universal-ctags/ctags.git'
+
+  option "without-iconv", "Enable multibyte character encoding"
```

Comment:
Right. I'm specifically asking about the help text that you've provided, though. Building *with* iconv would enable multibyte support, but if I do `brew info universal-ctags` with this addition, I would see that adding `--without-iconv` somehow enables multibyte character encoding, and I'm not sure that's correct.